packages/e2e_tests/cypress/e2e/examinaton-timetable-organisation-submission.cy.js

```diff
@@ -1,20 +1,18 @@
 import { BasePage } from '../pageObject/basePage';
 import { PO_ExaminationTimetable } from '../pageObject/Examination-TimeTable/PO_ExaminationTimetable';
-import { PO_ProjectPage } from '../pageObject/Search-and-project-pages/PO_ProjectPage';
 
 const examinationTimetable = new PO_ExaminationTimetable();
-const projectPage = new PO_ProjectPage();
 const basePage = new BasePage();
 
-before(() => {
-	cy.clearCookies();
-});
-
 describe('User registers as an organisation to have their say against the examination timetable', () => {
+	before(() => {
+		cy.clearCookies();
+		cy.navigateAndSearch('Front Office Auto Test');
```

Comment:
Is this going to work?  I checked on test and that search on project page didn't seem to do it.  May be wrong though.

Reply:
I added this method to all tests as we were navigating directly to the project page, which would not replicate a end to end journey. all tests have passed with this new method or are you specifically calling out this test?